man/systemd.exec.xml

```diff
@@ -89,6 +89,15 @@
 
     <variablelist class='unit-directives'>
 
+      <varlistentry>
+        <term><varname>ExecSearchPath=</varname></term>
+
+        <listitem><para>Takes a directory path relative to which the executable used by
```

Comment:
Your current implementation also allows multiple paths like `/foo:/bar`, since internally it just prepends to `PATH`. This should either be documented, or not permitted (probably just documented).

Reply:
this is evil. we should support multiple paths properly, and parse them/validate them properly, and expose them via the bus as array of strings rather than a single string